src/Compilers/Core/Portable/Symbols/IModuleSymbol.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
What does GetModuleNamespace return if that module doesn't have anything in the namespace?

Reply:
I ended up fixing this in a later commit from my branch. I'll amend this pull request to correct it from the start.

Reply:
Fixed. You can see the updates in https://github.com/dotnet/roslyn/compare/5b81932f59775bba34ebdcd741b404ed1651b04a..295cc8130785ca2b50a6d589ec0df3c8c6cfb504.